### DISCUSSION TITLE
is there a dynamic table ?

### DISCUSSION THREAD

**Discussion by iliassela** [myndpm/open-source#34] [Dynamic Forms]
hello, i want to ask if there a dynamic table, i found one but not custom, and i want a display mode like this : 
     Nom      :  MyName 
     Prenom  :  MyPrenom
     Age        :  20

**Comment by matheo** [comment]
Hi @iliassela 
The dynamic-editable-table is a plan for the future.
Recently I saw activity in this thread: https://github.com/angular/components/issues/5927
I hope you can find it useful
